src/main/java/duke/command/MainFunctions.java

```diff
@@ -0,0 +1,116 @@
+package duke.command;
+
+import duke.task.Deadline;
+import duke.Duke;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+import duke.exception.BlankDescException;
+import duke.exception.DukeException;
+
+public class MainFunctions {
+    public static final String LOGO =
+            "\t __   __\n"
+                    + "\t|  | |  |   ____     _  _   _    _\n"
+                    + "\t|  |_|  |  / _  \\   | |/_\\ | |  | |\n"
+                    + "\t|   _   | | |_|  \\  |  /   | \\_/  |\n"
+                    + "\t|__| |__|  \\___/\\_\\ |_|     \\__/|_|\n";
+    public static final String DIVIDER = "\t____________________________________________________________";
+    public static final String SPACER = "\t";
+
+    public static void printWelcome() {
+        System.out.println(DIVIDER);
+        System.out.println(LOGO);
+        System.out.println(SPACER + "Hello! I'm Haru");
+        System.out.println(SPACER + "What can I do for you?");
+        System.out.println(DIVIDER);
+    }
+
+    public static void printGoodbye() {
+        System.out.println(DIVIDER);
+        System.out.println(SPACER+"Bye. Hope to see you again soon!");
+        System.out.println(DIVIDER);
+    }
+
+    public static void invalidInput() {
+        System.out.println(DIVIDER);
```

Comment:
Method name sounds like a noun, maybe add a verb (eg. print) before to add clarity to its functionality?